# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/media-conduit/AudioConduit.cpp
# Commit: 549eae0865b6
# Full Hash: 549eae0865b6f00d030ad5af8551711ab3580133
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-09 10:01:40
# Regressor Bug: 1376873
# File Overlap Count: 1
# Description:
#   Bug 1376873 - Remove Android java code from AudioConduit; r=padenot
#   
#   This is no longer necessary as we do not use the webrtc.org audio device
#   implementations anymore.
#   
# ==============================================================================

diff -r 949c043fb183 -r 549eae0865b6 media/webrtc/signaling/src/media-conduit/AudioConduit.cpp
--- a/media/webrtc/signaling/src/media-conduit/AudioConduit.cpp	Mon Jul 16 11:50:40 2018 -0400
+++ b/media/webrtc/signaling/src/media-conduit/AudioConduit.cpp	Thu Jul 26 10:12:58 2018 -0400
@@ -374,16 +374,6 @@
   CSFLogDebug(LOGTAG,  "%s this=%p", __FUNCTION__, this);
   MOZ_ASSERT(NS_IsMainThread());
 
-#ifdef MOZ_WIDGET_ANDROID
-  JavaVM* jvm = mozilla::jni::GetVM();
-  jobject context = mozilla::AndroidBridge::Bridge()->GetGlobalContextRef();
-
-  if (webrtc::VoiceEngine::SetAndroidObjects(jvm, (void*)context) != 0) {
-    CSFLogError(LOGTAG, "%s Unable to set Android objects", __FUNCTION__);
-    return kMediaConduitSessionNotInited;
-  }
-#endif
-
   if(!(mPtrVoEBase = webrtc::VoEBase::GetInterface(GetVoiceEngine())))
   {
     CSFLogError(LOGTAG, "%s Unable to initialize VoEBase", __FUNCTION__);
